src/main/java/com/java_essential/Cherednichenko/classwork/lesson_2/Course.java

```diff
@@ -3,14 +3,12 @@
 import java.util.ArrayList;
 
 public class Course {
-    private ArrayList<Student> students;
+    private final ArrayList<Student> students=new ArrayList<>();
```

Comment:
```suggestion
    private final ArrayList<Student> students=new ArrayList<>();
```
change to List<Student> instead of ArrayList<Student> instance